QUICKSTART.md

```diff
@@ -401,11 +394,11 @@ in bulk. Multiple Span processors can be configured to be active at the same tim
 ```java
     SdkTracerProvider tracerProvider = SdkTracerProvider.builder()
       .addSpanProcessor(
-        MultiSpanProcessor.create(Arrays.asList(
+        SpanProcessor.composite(
```

Comment:
I guess multiple calls to `addSpanProcessor` would be more natural, and ends up with us creating the composite

Reply:
done